keps/sig-storage/625-csi-migration/README.md

```diff
@@ -194,12 +194,31 @@ you need any help or guidance.
 
 * **How can this feature be enabled / disabled in a live cluster?**
   - [x] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: CSIMigration, CSIMigration{cloud-provider}
+    - Feature gate name: CSIMigration, CSIMigration{cloud-provider}, InTreePlugin{cloud-provider}Unregister
     - Components depending on the feature gate: kubelet, kube-controller-manager, kube-scheduler
     - Please refer to this design doc on the [Step to enable the feature](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md#upgradedowngrade-migrateunmigrate-scenarios)
 
 * **Does enabling the feature change any default behavior?**
-  Yes and No. If only `CSIMigration` feature flag is enabled, nothing will change on the cluster behavior. However, if `CSIMigration` && `CSIMigration{cloud-provider}` are both enabled, the behavior will change. The in-tree volume plugin that the cloud-provider use will be redirect to use the corresponding CSI driver. But from a user perspective, nothing will be noticed.
+
+  Yes and No.
+  - If only `CSIMigration` feature flag is enabled, nothing will change on the cluster behavior. `CSIMigration`
+  is a big umbrella feature gate. `CSIMigration{cloud-provider}` will not take any affect without it.
+  - If only `CSIMigration{cloud-provider}` feature flag is enabled, nothing will change on the cluster behavior.
+  - `InTreePlugin{cloud-provider}Unregister` is a standalone feature gate that can be enabled and disabled
+  even out of CSI Migration scope. The name speaks for itself, when enabled, the controller will not 
+  register the specific in-tree storage plugin to the supported list. If User only enables this flag, they
+  will get an error from PVC saying it cannot find the plugin.
```

Comment:
You can clarify that a cluster operator may want to enable this regardless of csi migration if they do not want to support the legacy intree APIs and only support CSI going forward.